app/controllers/application.js

```diff
@@ -218,14 +218,16 @@ export default class ApplicationController extends ParachuteController {
 
     const amendmentsLayerDisabled = this.get('model').layerGroups.toArray().filter(layerGroup => layerGroup.get('visible')).find(layer => layer.id === 'amendments') === undefined;
 
+    const streetSectionLayerDisabled = this.get('model').layerGroups.toArray().filter(layerGroup => layerGroup.get('visible')).find(layer => layer.id === 'street-sections') === undefined;
+
     // Open the popup and clear its content (defaults to showing spinner)
     this.set('popupFeatures', null);
     this.set('popupLocation', e.lngLat);
 
     // Query and set the popup content
     const { lng, lat } = e.lngLat;
     const SQL = `
-    SELECT the_geom, 'alteration' AS type, altmappdf, status, effect_dt AS effective, NULL AS bbl, NULL AS address
+    SELECT the_geom, 'alteration' AS type, altmappdf, status, effect_dt as effective, NULL as bbl, NULL as address, NULL::timestamp as last_date, NULL as do_path
```

Comment:
It may be nice to consistently use the same case for SQL keywords, like 'AS'. I prefer uppercase because I think the SQL convention is to yell